### ISSUE TITLE
define( "WP_CACHE", true ); is not in wp-config.php. Either click "Attempt Fix" or add the code manually. Attempt Fix

### ISSUE THREAD

**Issue by githubsteef** [tlovett1/simple-cache#95]
define( "WP_CACHE", true ); is not in wp-config.php. Either click "Attempt Fix" or add the code manually.
Attempt Fix
have put in the code, still get this error

**Comment by stefanos82** [comment]
Yep, I can confirm this, along with the added `^M` characters that are visible in Vim editor.

I replaced the characters with `:%s/\r//g`, but it doesn't make any sense to repeat the same command over and over when I'm tweaking the cache's settings.

**Comment by ram108** [comment]
Yes. The plugin also breaks the `wp-config.php` file due to incorrect line breaks.

**Comment by nuevaya** [comment]
I confirm the plugin breaks the wp-config.php adding "extra" characters on every line of the file.

Please add the define( "WP_CACHE", true );  manually before activating the plugin

**Comment by MagneticArrow** [comment]
There is a typo in the backend message.  The actual code that plugin will write if able is:

define( 'WP_CACHE', true ); // Simple Cache

The typo is double quote appearing in WP backend ("WP_CACHE"), but really should be single quote.

I 444 my htaccess and wp-config.  And have the correct cache define in place, yet when I make a change in the plugin options, the message to add cache true appears again.  Should certain permissions exist for wp-config, or is once the cache define line is in place it is okay?
